### DISCUSSION TITLE
NextResponse not imported in middleware snippet for Nextjs Todo App

### DISCUSSION THREAD

**Discussion by akins-dev** [teamhanko/hanko#1089] [Q&A]
[link to docs](https://docs.hanko.io/tutorials/nextjs-todo)
Scroll to middleware to see the issue.
![Screenshot from 2023-10-13 02-22-18](https://github.com/teamhanko/hanko/assets/105003667/48cb6e04-2596-4b24-a2a1-5ee6e7c003e5)

Can someone make a PR to correct this?

**Comment by akins-dev** [comment]
I suggest a full revision of the [Nextjs Todo App docs](https://docs.hanko.io/tutorials/nextjs-todo). There are few updates to be made, especially with the code snippets.

**Comment by Ashutosh-Bhadauriya** [comment]
Hey @akins-dev can you please provide more details, where are you facing the issues?
